mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -68,6 +68,9 @@ service MobilecoindAPI {
     // Database encryption
     rpc SetDbPassword (SetDbPasswordRequest) returns (google.protobuf.Empty) {}
     rpc UnlockDb (UnlockDbRequest) returns (google.protobuf.Empty) {}
+
+    // Versioning
+    rpc GetReleaseVersion(google.protobuf.Empty) returns (MobilecoindVersionResponse) {}
```

Comment:
What does 'Release' version mean? Might `GetVersion` work well enough?

Reply:
Yea I guess 'Release' doesn't add any more information at this point. I'll change that thanks!